### DISCUSSION TITLE
kaç para olucak

### DISCUSSION THREAD

**Discussion by msdosqwe** [IBAX-io/go-ibax#1308] [Soru Cevap - Türkçe]
ilerde kaç para olur tahminleri alalım.

**Comment by Skips1** [comment]
Tamamen projeye bağlıdır ama benim pek bir beklentim yok arzı bayağı yüksek değeri 1 dolar olursa iyi bence yetkillierin stratejileriylede doğrudan orantılı herşey için çok erken bence bekleyip göreceğiz

**Comment by Yusufknk** [comment]
İlerleyen süreçlerde yakım olursa ki olacak, güzel yerlere gelecektir.

**Comment by OuzounE** [comment]
hodl arkadaslar yeni bir proje
